core/mgmt/v1beta/mgmt_public_service.proto

```diff
@@ -332,6 +332,16 @@ service MgmtPublicService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {tags: "Metric"};
   }
 
+  // List Instill Credit consumption time charts
+  //
+  // Returns a timeline of Instill Credit consumption for a given owner. The
+  // timeline consists of a list of time frames that contain the aggregated
+  // credit consumption.
+  rpc ListCreditConsumptionChartRecords(ListCreditConsumptionChartRecordsRequest) returns (ListCreditConsumptionChartRecordsResponse) {
```

Comment:
Hi @jvallesm 
Just to confirm, we will use a query parameter to choose the owner, right?